### ISSUE TITLE
side effects if session_start() in comments.php

### ISSUE THREAD

**Issue by gerritvanaaken** [Addpixel/KirbyComments#15]
In my template file I check if a user is logged, like this:

````
<?php if ($site->user() && $site->user()->role() == 'admin' ): ?>
  Hallo Admin!
<?php endif ?>
````

If do this before the comments include, I get the following error:
![bildschirmfoto 2017-02-27 um 23 07 56](https://cloud.githubusercontent.com/assets/227735/23382214/9de5294a-fd41-11e6-93b8-04578c2a3bf5.png)

If I do it after including the comments, the `$site->user()` is always false, even when I’m properly logged in. Can you do something about it?

**Comment by florianpircher** [comment]
Looks like line 103 in plugin/comments.php should be replaced by:

```php
if (session_status() == PHP_SESSION_NONE) {
    session_start();
}
```

I will include this in the next release.

**Comment by florianpircher** [comment]
This has been fixed by [v1.2.2](https://github.com/Addpixel/KirbyComments/releases/tag/1.2.2).
